microsoftgraph/microsoft-graph-explorer-v4
# Task: Remove .spec.tsx tests add tests on App.spec.tsx file

## Overview
Fixes #1763 

### Demo
Sample of the tests on App.spec.tsx file
<img width="422" alt="image" src="https://user-images.githubusercontent.com/45680252/173763632-59598607-911f-4583-bbf9-0978b430397b.png">


### Notes
Since we are testing what the end user sees and not implementation details, all necessary assertions have been made on the App.spec.tsx file. 
The unnecessary tests on the other .tsx files have been removed.
The tests confirm that all major sections of the app have been rendered, and that what we expect our users to see are available. 

## Testing Instructions

* How to test this PR
Check out this branch
Run 'npm run test'
Notice that all tests run in under 2 minutes
